support/cas-server-support-oauth-api/src/main/java/org/apereo/cas/ticket/refreshtoken/OAuth20RefreshToken.java

```diff
@@ -15,4 +18,22 @@
      * The prefix for refresh tokens.
      */
     String PREFIX = "RT";
+
+    /**
+     * Client id for whom this access token was issued.
+     *
+     * @return client id.
+     * @since 6.2
+     */
+    String getClientId();
+
+    /**
+     * Gets descendant OAuth access tokens.
+     *
+     * @return the descendant tickets
+     * @since 6.2
+     */
+    default Collection<String> getDescendantTickets() {
```

Comment:
Since we are only going to track access tokens for a refresh-token, it's best to simply rename this to getAccessTokens(). 
Also please change the return type to a Set.

Reply:
Instead of Collection ? Right ?

Reply:
Done